Scrapper-Service/test/mongodb_test.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from database import mdb
```

Comment:
* U can import class directly. 
* Test your logic for insertion failing on same data here.